modules/tester.js

```diff
@@ -1063,7 +1063,7 @@ Tester.prototype.begin = function begin() {
         config = getConfig([].slice.call(arguments)),
         next = function() {
             config.test(this, this.casper);
-            if (this.options.concise)
+            if (!this.options.concise)
```

Comment:
Perhaps curly brackets could be used here, as it seems to be the case with the rest of the file